package.json

```diff
@@ -45,6 +45,7 @@
     "react-measure": "^2.5.2",
     "react-router-dom": "^5.0.1",
     "react-scripts": "3.0.1",
+    "react-select": "^5.7.3",
```

Comment:
```suggestion
```

Reply:
Maybe we don't need this dependency anymore?

Reply:
no we do not - will remove